packages/lit-html/src/test/lit-html_test.ts

```diff
@@ -509,6 +511,15 @@ suite('lit-html', () => {
       );
       assert.equal(container.querySelector('span'), beforeRenderedNode);
     });
+
+    test('back-to-back expressions', () => {
```

Comment:
How is this test relevant? (probably just missed it)

Reply:
It tests a small fix added to `getTemplateHtml`